mmpose/datasets/pipelines/bottom_up_transform.py

```diff
@@ -165,8 +165,8 @@ def __call__(self, joints):
             tot = 0
             for idx, pt in enumerate(joints[i]):
                 x, y = int(pt[0]), int(pt[1])
-                if (pt[2] > 0 and x >= 0 and y >= 0 and x < self.output_res
-                        and y < self.output_res):
+                if (pt[2] > 0 and self.output_res > y >= 0
+                        and self.output_res > x >= 0):
```

Comment:
< is more natural,  `0 <= x < self.output_res`

Reply:
Modified.